app/Http/Controllers/AccountController.php

```diff
@@ -62,12 +63,7 @@ public function store()
 
             Auth::loginUsingId($user->id);
 
-            Mail::send('emails.newsignup', array('email' => Input::get('email')), function ($message) {
-                $message
-                    ->from('matt@symposiumapp.com', 'Matt Stauffer at Symposium')
-                    ->to(getenv('admin_email'), 'Admin')
-                    ->subject('New user signup on save my proposals');
-            });
+            Slack::send("*New user signup:*\n$user->first_name $user->last_name\n$user->email");
```

Comment:
I think it makes more sense to fire a regular event here that anyone can listen for, and create a single Slack event listener where we store all our Slack notifications so it's easy to see which events are going to Slack all in one place.

http://laravel.com/docs/5.0/events#event-subscribers

I'd say just use named events though instead of going full bore with classes for each. Dropping event classes for just event keys was like my favorite thing about Laravel coming from .NET, like zero benefit to creating an entire class for it.